webhook_to_fedora_messaging/endpoints/apikey.py

```diff
@@ -0,0 +1,86 @@
+from flask import Blueprint, Flask, request, Response, Request
+from ..database import db
+from ..models.apikey import APIKey
+from ..models.user import User
+from datetime import datetime
+from sqlalchemy_helpers import get_or_create
+from .util import not_found, success, bad_request, created, conflict, validate_request, exclude_from_val
+
+app = Flask(__name__)
+apikey_endpoint = Blueprint("apikey_endpoint", __name__)
+
+@apikey_endpoint.route("/apikey", methods=["POST"])
+def create_apikey():
+    """
+        Used for creating a new service by sending a post request to /service/ path.
+    """
+    
+    session = db.Session()
+    body = request.json
+    
+    
+    user = session.query(User).filter(User.username == body['username']).first()
```

Comment:
Add exception handling for `KeyError` (when the `username` key is not found in the JSON body). Abort the request with the `HTTP 429 Unprocessable Entity` error.